README.md

```diff
@@ -119,6 +119,30 @@ const HF_TOKEN = "hf_...";
 
 const inference = new HfInference(HF_TOKEN);
 
+// Chat completion API
+const out = await inference.chatCompletion({
+  model: "mistralai/Mistral-7B-Instruct-v0.2",
+  messages: [{ role: "user", content: "Complete the this sentence with words one plus one is equal " }],
+  max_tokens: 100
+});
+
+// Chat completion API on OpenAI (also compatible with MistralAI api, etc.)
+const openai = new HfInference(OPENAI_TOKEN).endpoint("https://api.openai.com");
+const out = await openai.chatCompletion({
+  model: "gpt-3.5-turbo",
+  messages: [{ role: "user", content: "Complete the this sentence with words one plus one is equal " }],
+  max_tokens: 100
+});
+
+// Streaming chat completion API
+for await (const chunk of openai.chatCompletionStream({
+  model: "gpt-3.5-turbo",
+  messages: [{ role: "user", content: "Complete the this sentence with words one plus one is equal " }],
+  max_tokens: 100
+})) {
+  console.log(chunk.choices[0].delta.content);
+}
```

Comment:
do we really need/want to support openai as a provider? in any case i dont think we want to document it in the main README

Reply:
great point, the main support is TGI and Serveless, but it's possible to use other providers, this could be very nice if your app, switch between local, cloud. But I agree it doesn't need to be on the main Readme, there is an snippet on the other doc.